auth/user_mgt_test.go

```diff
@@ -2051,9 +2195,10 @@ type mockAuthServer struct {
 // echoServer takes either a []byte or a string filename, or an object.
 //
 // echoServer returns a server whose client will reply with depending on the input type:
-//   * []byte: the []byte it got
-//   * object: the marshalled object, in []byte form
-//   * nil: "{}" empty json, in case we aren't interested in the returned value, just the marshalled request
+//   - []byte: the []byte it got
```

Comment:
I am curious why we need this change. Are rest of the comment blocks using - instead of *?

Reply:
Looks like it was auto generated. Reverting it to see if checks are still passing.

Reply:
Looks like the linter on my editor just saves like this. Doesn't make a difference to the code as there are no other itemized comment blocks.